src/use-themed-colormap.mjs

```diff
@@ -0,0 +1,9 @@
+import { useColorMode } from 'theme-ui'
+import useColormap from './use-colormap'
+
+const useThemedColormap = (name, options) => {
+	const [mode, ] = useColorMode()
```

Comment:
Hm I wonder why this isn't getting picked up by the linter? 🤔 

The code here looks good though!

Reply:
fixed!